rules/use-iapimarker-with-webapplicationfactory/rule.md

```diff
@@ -47,8 +47,19 @@ This approach means you don't need to do all the InternalsVisibleTo setup, but d
 
 The `IApiMarker` interface is a simple interface that is used to reference the application project.
 
+```cs
+namespace RulesApi;
+
+// This marker interface is required for functional testing using WebApplicationFactory.
+// For full explanation, see SSW Rule "Do you use IApiMarker with WebApplicationFactory"
+// https://www.ssw.com.au/rules/use-iapimarker-with-webapplicationfactory/
```

Comment:
```suggestion
// see https://www.ssw.com.au/rules/use-iapimarker-with-webapplicationfactory/
```

making it a little more terse and avoiding maintenance debt over the rule name

Reply:
Suggestion accepted with correction to capitalisation.